lib/webpagetest.js

```diff
@@ -474,7 +474,9 @@ function runTest(what, options, callback) {
 
   // poll results
   if (options.pollResults && !options.dryRun) {
-    options.pollResults = parseInt(options.pollResults * 1000, 10) || 5000;
+    // setting the lower limit to 20s
+    options.pollResults = Math.max(20, options.pollResults) * 1000;
+    options.pollResults = parseInt(options.pollResults, 10) || 60000;
```

Comment:
why d you need to specify the radix? also can you write this logic a bit better? why do you need parseInt?